lib/ProductOpener/Display.pm

```diff
@@ -246,6 +246,12 @@ The files that need to be checked need to be specified in the code of Display.pm
     	"js/dist/product-multilingual.js" => "JS file generated by the 'npm run build' command",
     );
 
+=head3 $knowledge_panels_options_ref
```

Comment:
This seems to be misplaced, was it a comment for an argument to a function?

Reply:
I put it (without the header) in init_request because it's something that's initialized there, and I think it's important to document.